src/app/components/core/notifications/add-notification-wizard/add-notification-wizard.component.ts

```diff
@@ -171,6 +172,7 @@ export class AddNotificationWizardComponent implements OnInit, OnDestroy {
                 this.pluginData.modalState = false;
                 this.selectInstalledPlugin();
               }
+              this.pluginRequestDone = true;
```

Comment:
1s timeout enough?

Reply:
No need to change timeout, only need to enable "Next" button after getting the response from `plugin` API.